scaling_performance/cluster_limits.adoc

```diff
@@ -51,6 +51,10 @@ version or storage data format.
 | 10,000
 | 10,000
 
+| Number of builds
+| N/A
+| 10,000
```

Comment:
If this is based only on the jenkins pipeline strategy build tests we recently did, this feels like a pretty generic claim @mrsiano .... our other build strategies would not influenced by the Jenkins constraints we were investigating.

Or am I missing some additional background here ?

@bparees fyi

Reply:
and our other build strategies would be much more subjected to the number of pods the cluster was capable of running since, unlike a pipeline build, "regular" builds need a pod for each build.

Reply:
right, so stage 1 was to make sure the Jenkins plugin core working as expected, so we narrow down other threats to identify the plugin capabilities.

what do you think about sperate the build capacity to two diffrent aspects as follow:
- Number of builds  runs by jenkins-sync-plugin.
- Number of builds

Ashley, could you suggest how to name it correctly ?

Reply:
Yep we need separated categories, minimally with the Pipeline Strategy Builds separated out (that is the name for what sync plugin handles).

I don't think we need further separation regarding capacity for the source/docker/custom strategies, but I'll defer to @bparees

Reply:
+1

Reply:
Based on Gabe's comments, perhaps we can label these as Pipeline Strategy Builds and Total Builds.

Reply:
Also @mrsiano - for the pipeline stuff, I would be inclined to also provide the following details
a) that the 10K number was achieved with 512Mi container limit and 256Mi JVM heap
b) and that when you doubled those number, you got 20 something K builds  (if I am remembering your findings correctly)

Thoughts?